client/components/SingleAlbum.js

```diff
@@ -1,64 +1,85 @@
 import React from 'react';
 import { connect  } from 'react-redux';
 import { renderSingleAlbum } from '../redux/albums/albumActions';
+import { renderReviews } from '../redux/reviews/reviewActions';
+import { AddReview } from './index';
 
 
 class SingleAlbum extends React.Component{
-  // constructor(props) {
-  //   super(props)
-  //   // this.handleAddToCart = this.handleAddToCart.bind(this)
-  // }
+  constructor() {
+    super()
+    this.state = {
+      review: '',
+      showHideReviewForm: false
+    }
+    this.showComponent = this.showComponent.bind(this);
+    // this.handleAddToCart = this.handleAddToCart.bind(this)
+  };
 
   componentDidMount() {
     const id = this.props.match.params.id;
```

Comment:
is this id an album id? we might want to be a bit more explicit in variable names that are somewhat common